cpp/src/plasma/test/serialization_tests.cc

```diff
@@ -353,13 +358,15 @@ TEST(PlasmaSerialization, WaitReply) {
   /* Create a map with two ObjectRequests in it. */
   ObjectRequestMap objects_in(num_objects_in);
   ObjectID id1 = ObjectID::from_random();
-  objects_in[id1] = ObjectRequest({id1, 0, ObjectStatus_Local});
```

Comment:
In this test case, the init value for the second argument is 0. However, in ObjectRequestType definition, there is no corresponding enum value to 0. I changed the value to ObjectRequestType::PLASMA_QUERY_LOCAL which equals to 1. The test has no problem. Will this change cause other problems? Maybe this is the benefit from scoped enum. We cannot assign arbitrary value to a enum type.